src/frontend/staff-portal/src/app/components/jj-dispute-assignments/jj-dispute-assignments.component.ts

```diff
@@ -42,21 +40,6 @@ export class JJDisputeAssignmentsComponent implements OnInit, AfterViewInit {
     "timeToPayReason",
   ];
 
-  // TODO dynamically get list of JJs
-  public jjList: JJTeamMember[] = [
-    { idir: "ldame@idir", name: "Lorraine Dame" },
-    { idir: "pbolduc@idir", name: "Phil Bolduc" },
-    { idir: "choban@idir", name: "Chris Hoban" },
-    { idir: "kneufeld@idir", name: "Kevin Neufeld" },
-    { idir: "rpress@idir", name: "Roberta Press" },
-    { idir: "cohiggins@idir", name: "Colm O'Higgins" },
-    { idir: "bkarahan@idir", name: "Burak Karahan" },
-    { idir: "twong@idir", name: "Tsunwai Wong" },
-    { idir: "ewong@idir", name: "Elaine Wong" },
-    { idir: "jmoffet@idir", name: "Jeffrey Moffet" },
-    { idir: "rpress@idir", name: "Roberta Press" },
-  ];
-
   constructor(
     public jjDisputeService: JJDisputeService,
```

Comment:
services are better to be private

Reply:
.. but then you can't call them from the html

Reply:
Ya, but it is not good to call from html directly